pkg/k8s/templates/scaledobject.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+  {{- range $key, $val := .Labels }}
+    {{ $key }}: {{ $val }}
+  {{- end }}
+spec:
+  minReplicaCount: {{ .MinReplicas }}
+  maxReplicaCount: {{ .MaxReplicas }}
+  cooldownPeriod: {{ .CooldownPeriod }}
+  pollingInterval: 1
+  scaleTargetRef:
+    name: {{ .DeploymentName }}
+    kind: Deployment
+  triggers:
+    - type: external-push
+      metadata:
+        scalerAddress: {{ $.ScalerAddress }}
+        hosts: '{{ join "," .Values.Hosts }}'
+  advanced:
+    restoreToOriginalReplicaCount: true
```

Comment:
What is this for? We don't use templates anymore since https://github.com/kedacore/http-add-on/pull/610.
Instead of using templates, we switched to use KEDA as dep

Reply:
Probably a confusion while doing the big rebasing. Thanks for catching it I'll remove it!